### PR TITLE
Docs: Improve test folder documentation

### PR DESCRIPTION
**Pull request by gziolo**
We merged the latest update of test folder documentation without paying too much attention. I would like to re-phrase it a bit and add some language corrections. I'm not a native speaker, so please let me know if there is still something that can be stated better :)

/cc @aduth @blowery @rralian

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
 ## Single test runner
 
 ### How to add a new test file
+The single test runner supports automatic test discovery. We only need to put a test file into a `test` subfolder, next to the files we want to test.
+The runner uses this glob pattern `@(client|server|test)/**/test/*.@(js|jsx)` to find test files.
 
-Single test runner supports automatic test discovery. You only need to put your test file into `test` subfolder next to file under test.
-Runner uses this `glob` pattern `@(client|server|test)/**/test/*.@(js|jsx)` to find test files.
+We should use the same file names as the implementation files for the tests.
+Example: if we want to cover with unit tests `hello-world/index.jsx` file, we should name a test file `hello-world/test/index.jsx`.
```
Comment (aduth) [inline test/README.md]: I'd rephrase this as:

>Example: if we want to write unit tests covering the file `hello-world/index.jsx`, we should name a test file `hello-world/test/index.jsx`.

File: test/README.md [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
+If we ever need to add non-test files to a `test` folder, we should put them in a deeper level. Common choices are:
 
-If we ever need to add non-test files to `test` folder we should put them on a deeper level. Common choices are:
 * `test/mocks/name.js` for test mocks
 * `test/fixtures/name.js` for test data
 
 ### How to run single test runner
 
-You can run all tests from the root folder using the `npm test` command.
+We have one command to run the tests `make test` from the root folder. It’s now an alias to the `npm test` command.
```
Comment (aduth) [inline test/README.md]: Should we favor one or the other, and drop mention of the alias? Thinking something like:

>Executing `make test` from the root folder will run all test suites. Behind the scenes [...]

File: test/README.md [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
+Behind the scenes we maintain 3 test runners. This is because every folder (`client`, `server` & `test`) have different `NODE_PATH` paths.
```
Comment (aduth) [inline test/README.md]: I'd change:

every folder have different paths -> each folder has a different path

**Comment by aduth** [comment]
This looks much improved. Consider the suggestions, but otherwise good to merge 👍

File: test/README.md (hunk 2 above)
Comment (gziolo) [inline test/README.md]: It's okey to skip an alias part, and mention well known execution command. We can change that in the future when we decide to migrate fully to npm scripts :)

**Comment by gziolo** [comment]
All comments addressed, thanks for help :)
